app/controllers/bibliographic_controller.rb

```diff
@@ -1,4 +1,4 @@
-class BibliographicController < ApplicationController
+class BibliographicController < ApplicationController # rubocop:disable Metrics/ClassLength
```

Comment:
I think we have to enable this again at the end of the class

Reply:
Or add it as an exclusion in .rubocop.yml

Reply:
Thank you! I did not know it will stay on by default. I've added the enable clause at the end.